xfce_lightdm_logout_login-leap15.0.json

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
Please keep the timestamp in the filename of the needle as we have as an convention for - I hope - all of our needles. Unfortunately we do not yet have a check in this needle repo for this. So e.g. name it "xfce_lightdm_logout_login-leap15.0-20180525.json" and the .png accordingly